src/Aggregation/DateHistogramAggregation.php

```diff
@@ -45,9 +50,16 @@ protected function getType(): string
 
     protected function buildAggregation(): array
     {
-        return [
+        $build = [
             'field' => $this->field,
             'calendar_interval' => $this->calendarInterval,
         ];
+
+        if(!is_null($this->min) && !is_null($this->max)){
```

Comment:
can you replace `!is_null()` by `null !==`. I will enforce that with a CS rule :)

Reply:
Sure